portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro

```diff
@@ -557,6 +557,28 @@ definition {
 			locator1 = "PageEditor#EXPERIENCE_SELECT_ITEM");
 	}
 
+	macro addExperienceWithoutSegment {
```

Comment:
In this macro you are not adding any experience, you are trying to add an experience with no name. I think the best way is to modify the addExperience macro and add a condition to check what happens if the name is empty.